soh/soh/Enhancements/controls/GameControlEditor.cpp

```diff
@@ -252,19 +251,21 @@ namespace GameControlEditor {
                                                 "gFirstPersonCameraSensitivityX", 0.01f, 5.0f, "", 1.0f, true);
             UIWidgets::EnhancementSliderFloat("Aiming/First-Person Vertical Sensitivity: %d %%", "##FirstPersonSensitivity Vertical",
                                               "gFirstPersonCameraSensitivityY", 0.01f, 5.0f, "", 1.0f, true);
+        } else {
+            CVarSetFloat("gFirstPersonCameraSensitivity", 1.0f);
         }
         UIWidgets::Spacer(0);
-        LUS::EndGroupPanel();
+        LUS::Context::GetInstance()->GetWindow()->GetGui()->EndGroupPanel(0);
 
         UIWidgets::Spacer(0);
-        LUS::BeginGroupPanel("Third-Person Camera", ImGui::GetContentRegionAvail());
+        LUS::Context::GetInstance()->GetWindow()->GetGui()->BeginGroupPanel("Third-Person Camera", ImGui::GetContentRegionAvail());
```

Comment:
@Kenix3 Does this really belong in LUS anyway? (along with `EndGroupPanel`

Reply:
both `BeginGroupPanel` and `EndGroupPanel` are decently big, so it seems like we should definitely have them *somewhere*. any better spots come to mind?

Reply:
I think most UI helpers are in UIWidgets, that probably would make sense, but admittedly I haven't peeped the source of this one

Reply:
I think it'd make sense to move this code to UI Widgets.

I'd also like to eventually promote UIWidgets out of SoH and into LUS.